style.css

```diff
@@ -0,0 +1,640 @@
+  text-shadow: 0 0 20px rgb(98, 143, 135);
+}
+
+.header-block__text__h1 {
+  font-size: 20vw;
+}
+
+.header-block__text__p {
+  text-decoration: line-through 2px;
+  font-size: 5vw;
+  margin: -4% 0 0 40%;
+  letter-spacing: 20px;
+}
+
+
+
+
+
+
+
+/* FORM & ACCORDIAN */
+
+
+
+/* FORM */
+
+.fieldset {
+  margin: 1% 0% 0 1%;
+  
+  position: relative;
+}
+
+#form__fieldset{
+border: none;
+margin: 1%;
+}
+
+
+.radioallitems {
+  position: relative;
+  height: 100%;
+  width: 120%;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  margin: 1% 0 0% 2%;
+
+  overflow: hidden;
+}
+
+.radiocontainer {
+  height: 100%;
+  width: 100%;
+  display: block;
+  overflow: hidden;
+  margin-bottom: 0.7%;
+ 
+}
+
+.containerradio input {
+  opacity: 0;
+  cursor: pointer;
+  height: 10%;
+  width: 10%;
+  margin: 2% 0 2.9% 0;
+}
+
+
+/* Radio buttons */
+
+.radiobtn {
+  position: absolute;
+  left: 0;
+  height: 12%;
+  width: 1.8%;
+  background-color: rgba(100, 99, 99, 0.651);
+  border-radius: 50%;
+  margin: 0.8% 0% 0% 0.1%;
+ 
+}
+
+.containerradio:hover input ~ .radiobtn {
+  background-color: rgb(90, 86, 86);
+  box-shadow: 0 0 2px black;
+}
+
+.containerradio input:checked ~ .radiobtn {
+  background-color: #359273a8;
+}
+
+.radiobtn:after {
+  content: "";
+  position: absolute;
+  display: none;
+}
+
+.containerradio input:checked ~ .radiobtn:after {
+  display: block;
+}
+
+.containerradio .radiobtn:after {
+  margin: 19% 21%;
+  height: 60%;
+  width: 56%;
+  border-radius: 60%;
+  background: rgb(147, 226, 206);
+}
+
+.radio-btn-text {
+  font-size: 1em;
+  position: absolute;
+
+  margin: -5% 0% 0% 3%;
+}
+
+
+/* form - contact information */
+
+.form-div {
+  background-color: rgba(255, 255, 255, 0.39);
+  grid-area: form;
+  color: rgb(20, 58, 52);
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.signup-form {
+  display: block;
+  overflow: hidden;
+  margin: 1vw 10vw;
+  backdrop-filter: blur(10px);
+  background-color: rgba(219, 235, 231, 0.671);
+}
+
+.form-text {
+  font-weight: bolder;
+  font-family: "Lato", sans-serif;
+}
+
+.text-form {
+  margin: 0% 0 0% 2%;
+  font-size: 5em;
+}
+
+.contact-information {
+  margin: 2% 0% 0% 2%;
+}
+
+input[type="text"],
+[type="email"],
+[type="tel"],
+[type="password"] {
+  align-content: center;
+  width: 50%;
+  display: block;
+  margin-top: 0.5%;
+  padding: 1%;
+  color: rgb(24, 63, 56);
+  font-weight: bold;
+  border: 1px solid rgb(9, 31, 27);
+  background-color: rgba(163, 192, 187, 0.37);
+}
+
+input:valid {
+  background-color: rgba(60, 158, 112, 0.336);
+}
+
+input:invalid {
+  background-color: rgba(158, 60, 60, 0.226);
+}
+
+input:focus {
+  outline: none;
+  border: 3px dotted rgb(100, 100, 102);
+  color: rgb(22, 56, 52);
+  font-weight: bolder;
+}
+
+
+/* Checkbox - did try a diffrent way to style these (used a other method on the radio-btn)*/
+
+input[type="checkbox"] {
```

Comment:
I learn a lot while reading your code. :) I didn't have time to style my checkbox, but now I see how you solved it, thanks!